gradle.properties

```
@@ -22,18 +22,18 @@ mockitoVersion=3.7.7
 jacocoVersion=0.8.6
 wso2OrbitAntlrVersion=4.5.1.wso2v1
 ballerinaGradlePluginVersion=1.0.0
-bouncycastleVersion=1.69
+bouncycastleVersion=1.74
 wso2CommonsPoolVersion=1.5.6.wso2v1
 
 #stdlib dependencies
 stdlibConstraintVersion=1.1.0
 stdlibCryptoVersion=2.3.0
 stdlibIoVersion=1.4.0
 stdlibLogVersion=2.6.0
-stdlibHttpVersion=2.6.0
+stdlibHttpVersion=2.6.2-20231013-151500-b205e1d
```

Update http version after release
```suggestion
stdlibHttpVersion=2.6.2
```